content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+---
+title: Converting an existing Gatsby project to TinaCMS
+last_edited: '2024-10-25T08:29:23.867Z'
+next: content/docs/cli-overview.mdx
+previous: content/docs/contextual-editing/overview.mdx
+---
+
+> ⚠️ Warning: TinaCMS does not officially support Gatsby. We recommend migrating your Gatsby site to a well supported framework such as [Next.JS](https://tina.io/docs/frameworks/next/overview/) instead.
+
+## Introduction
+
+In this tutorial we'll be showing you how you might go about converting an existing gatsby-mdx blog to use TinaCMS. We've created a starter repo for you to follow along with. The repo is a fork of the official [Gatsby md blog starter](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-blog).
+
+## Limitations
+
+There are a few limitations with the approach outlined in this guide.
+
+- You don't get visual editing
+- You lose out in Gatsby's image optimization
+- Gatsby uses Github Flavoured Markdown, which TinaCMS doesn't fully support
```

Comment:
Ok, I'll commit this. But only because TinaCMS is probably geared towards American clients.

Also what's the rationale behind changing `doesn't` to `does not`? I thought we were trying to make the guide less verbose.